hr_course/models/hr_course_schedule.py

```diff
@@ -70,6 +75,57 @@ class HrCourseSchedule(models.Model):
         states={"in_validation": [("readonly", False)]},
     )
 
+    alerted = fields.Boolean()
```

Comment:
this a bit too vague for a name. Proposition
```suggestion
    course_expiration_alert_sent = fields.Boolean()
```